### ISSUE TITLE
January 4, 1970

### ISSUE THREAD

**Issue by dkim-io** [nicolo-ribaudo/babel-preset-current-node-syntax#6]
When I installed the package, some of the files have a timestamp of January 4, 1970 9:38 AM.

Namely files: LICENSE, README.md, src/index.js

Strangely this causes an error for us when we try to zip our code.

Are you experiencing something similar?

**Comment by agos-jwb** [comment]
We are also experiencing the same problem. ZIP file format does not allow dates before 1980. It is causing our AWS deployments to fail.

**Comment by nicolo-ribaudo** [comment]
Do you know what could have caused this problem and if it's something that I can fix?

**Comment by dkim-io** [comment]
What do you use to push the package to the NPM registry?

**Comment by nicolo-ribaudo** [comment]
I just run `lerna publish` with yarn 2. I'll check if other packages released with this combination have the same problem.

**Comment by agos-jwb** [comment]
We are using python zip. But it really does not matter what we would use to zip files. The Zip file format does not allow dates before 1980.

When you built the NPM package somehow those files got a date of January 4, 1970 9:38 AM.

A quick fix for us is to touch those files. I have modified our scripts to touch those files before we try and zip them. But I have to descend to all node_modules directories and we have dozens.

**Comment by shaunthegeek** [comment]
me too. https://github.com/aws/aws-cli/issues/2639

```bash
$ find ./node_modules/ -type f -printf '%T+ %p\n' | sort | head -n 10
1970-01-04+23:38:52.0000000000 ./node_modules/babel-preset-current-node-syntax/LICENSE
1970-01-04+23:38:52.0000000000 ./node_modules/babel-preset-current-node-syntax/README.md
1970-01-04+23:38:52.0000000000 ./node_modules/babel-preset-current-node-syntax/src/index.js
1985-10-26+16:15:00.0000000000 ./node_modules/@babel/code-frame/LICENSE
1985-10-26+16:15:00.0000000000 ./node_modules/@babel/code-frame/README.md
```

HACK:

```bash
find ./node_modules/babel-preset-current-node-syntax/ -mtime +10950 -print -exec touch {} \;
```

**Comment by nicolo-ribaudo** [comment]
Could you check if version `0.1.4-0` (`babel-preset-current-node-syntax@next`) solves the issue? If it does, I can republish it as a `0.1.4`.

**Comment by shaunthegeek** [comment]
@nicolo-ribaudo I have try it, always wrong:

```bash
$ rm -rf node_modules/babel-preset-current-node-syntax
$ npm install babel-preset-current-node-syntax@next
$ find ./node_modules/babel-preset-current-node-syntax/ -type f -printf '%T+ %p\n' | sort | head -n 10
1970-01-04+23:38:52.0000000000 ./node_modules/babel-preset-current-node-syntax/LICENSE
1970-01-04+23:38:52.0000000000 ./node_modules/babel-preset-current-node-syntax/README.md
1970-01-04+23:38:52.0000000000 ./node_modules/babel-preset-current-node-syntax/src/index.js
2020-09-27+00:17:18.7540922800 ./node_modules/babel-preset-current-node-syntax/package.json
```

**Comment by nicolo-ribaudo** [comment]
Thanks for the workaround, I have isolated this to a yarn bug: https://github.com/yarnpkg/berry/issues/1882

**Comment by nicolo-ribaudo** [comment]
This should be fixed in v0.1.4
